CMakeLists.txt

```diff
@@ -150,19 +157,38 @@ add_dependencies(examples examples_by_runtime_api_module modified_cuda_samples)
 
 install(
 	TARGETS cuda-api-wrappers
-	EXPORT cuda-api-wrappers
+	EXPORT cuda-api-wrappersExport
 	RUNTIME DESTINATION bin
 	ARCHIVE DESTINATION lib
 	LIBRARY DESTINATION lib
 	INCLUDES DESTINATION include
-	CONFIGURATIONS Release RelWithDebugInfo
 )
 
 install(
-#	EXPORT cuda-api-wrappers
 	DIRECTORY src/cuda
 	DESTINATION include
 	FILES_MATCHING REGEX "\\.(h|hpp|cuh)"
 )
 
-export(EXPORT cuda-api-wrappers FILE cuda-api-wrappers.cmake)
+install(
+	EXPORT ${PROJECT_NAME}Export
```

Comment:
seems like we're setting the export names twice in different ways

Reply:
The `export` function creates a build tree dependent export of the targets, whereas the `install` of the export creates a build tree independent export.

See https://github.com/codecircuit/cuda-api-wrappers/commit/ebb23908c5b1f8ff164ef0325d3cd633348db124

Thus the `export` function only makes sense if we export the build directory into the local CMake package registry. And then it only makes sense if the Export file is called `cuda-api-wrappersConfig.cmake` or `cuda-api-wrappers-config.cmake`, because these are the names CMake is searching for.

Reply:
```suggestion
	EXPORT ${PROJECT_NAME}_export
```